### DISCUSSION TITLE
Unexpected token '<', "<!DOCTYPE "... is not valid JSON in using remote storage

### DISCUSSION THREAD

**Discussion by raktimproloy** [GrapesJS/grapesjs#5489] [Q&A]
const editor = grapesjs.init({
    container: "#editor",
    blockManager: {
      appendTo: "#blocks",
    },
    styleManager: styleManager,
    layerManager: layerManager,
    traitManager: traitManager,
    selectorManager: selectorManager,
    panels: panels,
    deviceManager: deviceManager,
    assetManager: { assets: assets, autosave: false }, // 'upload' option has been removed
    storageManager: storageSetting(pageId),
    canvas: {
      styles: styles,
      scripts: scripts,
    },
    plugins: [
      tailwindComponent,
      gjsBlockBasic,
      swiperComponent,
      // grapesjsBlockBootstrap,
      grapesjsPluginExport,
      grapesjsStyleBg,
      chartLibComponent,
      grapesjsNavbar,
      // grapesjsParserPostcss
    ],
  });
  
  <######## Storage Setting #########>

  export const storageSetting = (pageId) => {
  const projectApi = `${API_HOST}api/pages/${pageId}/content`
  return {
    type: "remote",
    stepsBeforeSave: 1,
    contentTypeJson: true,
    storeComponents: true,
    storeStyles: true,
    storeHtml: true,
    storeCss: true,
    headers: {
      "Content-Type": "application/json",
      "accept": "application/json"
    },
    id: "mycustom_",
    urlStore: projectApi,
    urlLoad: projectApi,
  };
  
};

Note: When I use local storage I don't get any error..
![Screenshot_12](https://github.com/GrapesJS/grapesjs/assets/83780881/28623e79-7369-4cfb-b389-f21a8921101e)

**Comment by artf** [comment]
Debug your BE endpoint, you're clearly returning an HTML string instead of a JSON....
